src/settings-ui/Settings.UI/MainWindow.xaml

```diff
@@ -1,14 +1,15 @@
-﻿<Window
+﻿<winuiex:WindowEx
```

Comment:
Why do we need third-party package(s) again? Note: _afaik_ the current version of this, is quite outdated

Reply:
WinUIEx provides a lot of useful extension methods for the WinUI3 Window.
It's already used in PT settings and other utilities.

Reply:
Well, yes,but I want to ask with more emphasis on the **need**. I rather use native bits.

Reply:
WinUIEx is used throughout PT such as in the flyout window. Is it worth using if none of the extension methods are actively being used? WinUIEx.WindowEx seems to cause a few issues related to theming as well, since `WindowEx.RequestedTheme` does not exist like it does with `Window.RequestedTheme`.

I was looking at refactoring the flyout window in order to not use it since it complicates the theme there too, but will wait for the verdict on whether or not it is going to be supported by PT.

Reply:
@Jay-o-Way @BLM16 

Setting MinWidth and Mica is much easier with WinUIEx. The latter will be resolved with the next WinUI version that should be imminent, so it might be a good moment to moclve away.

Have we tested setting the RequestedTheme on a plain Window would give a diferent result vs.a WinUIEx window?

Btw, we should probably discuss this in a dedicated issue or PR :)

Reply:
This is exactly what I tried to push with https://github.com/microsoft/PowerToys/issues/20129, but it costs nothing make a new issue. I wish we **could** create an actual [Discussion](https://github.com/microsoft/PowerToys/issues/22185).

Reply:
Giving my two cents on this since I have worked on https://github.com/microsoft/PowerToys/issues/20129
That PR was a massive cleanup of unused dependency. At that time WinUIEx was already widely used across the solution.
Why do we want to get rid of that dependency? Are there any bugs? If yes, can you please open an issue?

Reply:
More like a matter of principle. I just hate the idea of being dependant on a third-party {something}. Especially since we're making something under the flag of Microsoft. Not even mentioning the fact it's outdated. (or did I?)

Reply:
I guess moving away from WinUIEx might make more sense on another future PR. It's already used by the project at this point.

Reply:
I agree. Having inconsistent UI libraries makes maintenance more difficult and requires contributors to learn multiple frameworks potentially. WinUIEx resolves some problems that are found in the native library, however it also poses some of its own.

Definitely a larger scale refactoring that should be done in a separate PR if desired.